drivers/spi/spi_context.h

```diff
@@ -183,6 +196,28 @@ gpio_dt_flags_t spi_context_cs_active_level(struct spi_context *ctx)
 	return GPIO_ACTIVE_LOW;
 }
 
+static inline int spi_cs_configure(const struct spi_cs_control *cs_gpio)
```

Comment:
The naming of these functions is misleading to me, they are all `_configure` but they only configure in one direction, unlike the GPIO functions for example. Perhaps `initialize`?